pkg/workspace/validate_test.go

```diff
@@ -236,3 +243,14 @@ func TestValidateOnlyOnePVCIsUsed_Invalid(t *testing.T) {
 		})
 	}
 }
+
+func getContextBasedOnFeatureFlag(featureFlag string) context.Context {
```

Comment:
Hmm this function keeps being copied around. Might want a function that takes in a context and returns another one with Alpha set e.g. withAlphaEnabled(ctx)

Reply:
(could be in a different PR)